### ISSUE TITLE
npu

### ISSUE THREAD

**Issue by brownplayer** [intel/AI-Playground#33]
It would be even better if later versions could add support for npu modules

**Comment by 3ae3ae** [comment]
I feel the same way: there isn't much software that utilizes Intel's NPUs.

**Comment by brownplayer** [comment]
Yes, for the time being, only openvino and the camera that comes with the notebook support npu.

---- Replied Message ----
| From | ***@***.***> |
| Date | 08/21/2024 10:16 |
| To | intel/AI-Playground ***@***.***> |
| Cc | brownplayer ***@***.***>,
Author ***@***.***> |
| Subject | Re: [intel/AI-Playground] npu (Issue #33) |

I feel the same way: there isn't much software that utilizes Intel's NPUs.

**Comment by 54shitaimzf** [comment]
You can try the plugin for Gimp or Audacity

**Comment by brownplayer** [comment]
> This software do update slow

Yes, it was updated too slowly, a little behind the model iterations
